gammapy/utils/modeling.py

```diff
@@ -289,9 +304,13 @@ def set_parameter_errors(self, errors):
         errors : dict of `~astropy.units.Quantity`
             Dict of parameter errors.
         """
+        for key in errors.keys():
```

Comment:
Remove `.keys()`?
If it's a dict, then for key in error will loop over the keys and is the normal way to write it.